pkgs/development/tools/avr-sim/default.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  lazarus,
+  fpc,
+  pango,
+  cairo,
+  glib,
+  atk,
+  gtk2,
+  libX11,
+  gdk-pixbuf,
+}:
+stdenv.mkDerivation rec {
+  pname = "avr-sim";
+  version = "2.8";
+
+  src = fetchzip {
+    url = "http://www.avr-asm-tutorial.net/avr_sim/28/avr_sim_28_lin_src.zip";
+    sha256 = "sha256-7MgUzMs+l+3RVUbORAWyU1OUpgrKIeWhS+ObgRJtOHc=";
+  };
+
+  nativeBuildInputs = [lazarus fpc];
+
+  buildInputs = [pango cairo glib atk gtk2 libX11 gdk-pixbuf];
+
+  NIX_LDFLAGS = "--as-needed -rpath ${lib.makeLibraryPath buildInputs}";
+
+  buildPhase = ''
+    runHook preBuild
+
+    lazbuild --lazarusdir=${lazarus}/share/lazarus --build-mode=Release avr_sim.lpi
+
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+
+    cp avr_sim $out/bin
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "AVR assembler simulator for the stepwise execution of assembler source code - with many extras.";
```

Comment:
Nit: meta.description should not end with a dot.

Reply:
Fixed.